pages/Ore-Washer.md

```diff
@@ -0,0 +1,14 @@
+The Ore Washer is in the [Basic Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machines) category. It is a multiblock structure.<br>
+
+## Obtaining
+The Ore Washer is a multiblock structure, it consists of a cauldron, a fence and a dispenser.<br>
+**Note:** It doesn't matter which side the dispenser is facing, place it to your liking.<br>
+
+A complete setup should look like this:
+
+![Armor Forge Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-ore-washer.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the Dispenser to collect your crafted armor.
```

Comment:
The Ore Washer does not create armor...

Reply:
Yeah I fixed that as soon as I pushed it lol